docs/deploying/azure.md

```diff
@@ -0,0 +1,81 @@
+---
+permalink: /docs/deploying/azure/index.html
+layout: docs
+---
+
+If you've been following along with [Getting Started](../README.md), it's time to deploy so you can use it beyond just your local machine.
+[Azure](http://http://azure.microsoft.com/) is a somewhat easy and probably unsupported way to deploy hubot.
```

Comment:
If it has documentation in this repository and website (which pulls from here), it's hard to call it unsupported.